website/docs/r/compute_instance_volume.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_volume"
+sidebar_current: "docs-scaleway-resource-compute-instance-volume"
+description: |-
+  Manages Scaleway Compute Instance Volumes.
+---
+
+# scaleway_compute_instance_volume
+
+Creates and manages Scaleway Compute Instance Volumes. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#volumes-7e8a39).
+
+## Example Usage
```

Comment:
Maybe we could add another example with a server attachment?

Reply:
Wouldn't it make more sense to link to the server doc? That way we only have to maintain one page when things change...